go.mod

```diff
@@ -190,8 +190,8 @@ require (
 	github.com/mailru/easyjson v0.7.7 // indirect
 	github.com/matttproud/golang_protobuf_extensions v1.0.2 // indirect
 	github.com/mistifyio/go-zfs v2.1.2-0.20190413222219-f784269be439+incompatible // indirect
-	github.com/mitchellh/go-wordwrap v1.0.0 // indirect
-	github.com/mitchellh/mapstructure v1.4.1 // indirect
```

Comment:
rather than updating mapstructure, I think we can drop it entirely (we only use it in kube-openapi, opened https://github.com/kubernetes/kubernetes/pull/118014 and https://github.com/kubernetes/kube-openapi/pull/390 to drop it)